### DISCUSSION TITLE
The country provided is not allowed

### DISCUSSION THREAD

**Discussion by shahid-0** [vercel/vercel#9109] [Help] [answered]
Is Vercel banned in Pakistan, when I tried to sign up with my GitHub then it wants a mobile number but when I enter my phone number and click on continue it will give that error, One of my friends also created an account on Vercel through Pakistan number but they didn't get any Error.

![image](https://user-images.githubusercontent.com/81429750/208739871-7e8133c0-93ba-411e-87a3-e2c9106e13ea.png)

**Comment by amyegan** [comment, marked as answer]
Sorry you're getting this error @shahid-0. 😞 Can you contact Vercel Support by emailing [registration@vercel.com](mailto:registration@vercel.com)? The team will be able to help you via email so your personal contact info doesn't get shared publicly like it would here.

**Comment by svibees** [comment]
I send the email but I did not get any reply yet
